help/guest-users.md

```diff
@@ -42,6 +42,37 @@ pricing](/help/zulip-cloud-billing#temporary-users-and-guests) for guest users.
 
 {end_tabs}
 
+## Configure whether guests can see all other users
+
+{!cloud-plus-only.md!}
+
+You can restrict guests' ability to see other users in the organization. If you
+do so, guests will be able to see information about other users only in the
+following cases:
+
+- The user belongs to a [direct message](/help/direct-messages) thread with the
+  guest.
+- The user is subscribed to the same [stream](/help/streams-and-topics) as the
+  guest.
```

Comment:
Would prefer to avoid the possible confusion that guests can only be on one stream. Also I think spelling out the DM examples might be helpful just to be super explicit: 
* The user has sent the guest a direct message or is a recipient of a direct message conversation including the guest.
* The user shares a stream subscription with the guest.

Reply:
I don't really find either of those to be more clear than what's written...

Reply:
Maybe this?
> The user is subscribed to one or more [streams](/help/streams-and-topics) with the guest.